pkg/webhook/validation/dynakube/oneagent_test.go

```diff
@@ -424,3 +424,41 @@ func createDynakubeWithHostGroup(args []string, hostGroup string) *dynatracev1be
 		},
 	}
 }
+
+func TestValidateOneAgentVersionIsSemVer(t *testing.T) {
+	testCasesAcceptedVersions := []string{"", "1.0.0", "1.200.1"}
+
+	testCasesNotAcceptedVersions := []string{"latest", "raw", "1.200.1-raw"}
```

Comment:
```suggestion
	testCasesNotAcceptedVersions := []string{"latest", "raw", "1.200.1-raw", "1.200.1+build", "1.200", "1", "1.0", "v1.200.0"}
```

Reply:
fixed in 8784b8194be7c36fec3bafd5de700098203ad796